ebml_matroska.xml

```diff
@@ -215,10 +215,10 @@
     <documentation lang="en">Set if the track is usable. (1 bit)</documentation>
   </element>
   <element name="FlagDefault" path="1*1(\Segment\Tracks\TrackEntry\FlagDefault)" cppname="TrackFlagDefault" id="0x88" type="uinteger" minOccurs="1" maxOccurs="1" minver="1" default="1" range="0-1">
-    <documentation lang="en">Set if that track (audio, video or subs) SHOULD be active if no language found matches the user preference. (1 bit)</documentation>
+    <documentation lang="en">For audio or video, set if that track SHOULD be active if no language found matches the user's preference; this can be used to specify the original content language. For subtitles, set if that track SHOULD be chosen if there are multiple tracks which match the user's language preferences. (1 bit)</documentation>
   </element>
   <element name="FlagForced" path="1*1(\Segment\Tracks\TrackEntry\FlagForced)" cppname="TrackFlagForced" id="0x55AA" type="uinteger" minOccurs="1" maxOccurs="1" minver="1" default="0" range="0-1">
-    <documentation lang="en">Set if that track MUST be active during playback. There can be many forced track for a kind (audio, video or subs), the player SHOULD select the one which language matches the user preference or the default + forced track. Overlay MAY happen between a forced and non-forced track of the same kind. (1 bit)</documentation>
+    <documentation lang="en">Applies only to subtitles. Set if that track SHOULD be active during playback by default if it matches the user's language preference, even if the user's preferences would normally not enable subtitles with the selected audio track; this can be used for tracks containing only translations of foreign-language audio or onscreen text. (1 bit)</documentation>
```

Comment:
No, this can't be subtitle only. That's the most common case. But we don't want to limit the format. For example you could have a forced video track. Some parts of the main video would be overlayed to show signs in the local language or censor parts in a specific country.

Reply:
But if the specs says now *only* subtitles can be forced. It will never ever be possible to force other types of tracks because it will break backward compatibility. That's why the specs are kept open on purpose.

Reply:
My issue is that the spec doesn't define anything useful that should happen when there's a forced audio or video track (I know it currently says "overlay MAY happen" but that clause might as well not exist seeing as nothing implements it).
Maybe we could say that authoring applications currently MUST not set this flag to 1 for video and audio tracks, and that player applications SHOULD ignore it, while leaving open the possibility that some meaning could be assigned to it in the future? (I'm just not sure what you would use it for that wouldn't be better-served by adding a new flag that doesn't have a the historical baggage.)